wallu/wallu.ino

```diff
@@ -60,6 +61,28 @@ int calculate_battery(int read_in)
   return 100.0 * (raw_voltage - BATT_MIN) / (BATT_MAX - BATT_MIN);
 }
 
+void set_eye_pattern(String pattern)
```

Comment:
Not saying we should do this for this PR, but it would be cool to have a class for the eyes too.